index.html

```diff
@@ -18,14 +25,15 @@
         </div>
         <div class="form-field">
           <label for="level" class="form-label">Level*</label>
-          <select name="level" class="form-input">
+          <select id="Level" name="level" class="form-input">
```

Comment:
- Excellent job linking the CSS and JavaScript files correctly.
- The id for the select element was changed to `Level` which should be in lowercase for consistency (`level`).